src/modules/Experience/scenes/AboutScene/components/MovingFace/MovingFace.tsx

```diff
@@ -45,13 +49,62 @@ const MovingFace = ({ scenePositionY, selectedColor }: MovingFaceProps) => {
       faceRef.current.rotation.y = mouseX * rotationIntensityY;
       //faceRef.current.lookAt(mouseX * lookAtIntensity, mouseY * lookAtIntensity, 20);
     }
+
   });
 
+  const map1 = new THREE.TextureLoader().load('https://cdn.discordapp.com/attachments/941095160517894185/1192890009741709403/note.png?ex=65aab865&is=65984365&hm=7c54f001dd572a6fc963abc396026353c22b73504b0ebfc96f6a6eac8df1d641&')
+  const model = useGLTF('/models/head-2.glb')
+  let uniforms = { mousePos: { value: new THREE.Vector3() } }
+  const cursor = { x: 0, y: 0 }
+  // Let's ignore the issue for now
+  // @ts-ignore
+  const modelGeo = model.scene.children[0].geometry.clone()
+  const pmaterial = new THREE.PointsMaterial({
+    color: new THREE.Color(`${selectedColor}`),
+    size: 0.1,
+    blending: THREE.AdditiveBlending,
+    transparent: true,
+    opacity: 1,
+    depthWrite: false,
+    sizeAttenuation: true,
+    alphaMap: map1
+  })
+
+  // Create the custom vertex shader injection
+  pmaterial.onBeforeCompile = function(shader) {
+    shader.uniforms.mousePos = uniforms.mousePos
+
+    shader.vertexShader = `
+      uniform vec3 mousePos;
+      varying float vNormal;
+
+      ${shader.vertexShader}`.replace(
+        `#include <begin_vertex>`,
+        `#include <begin_vertex>
+        vec3 seg = (position * -.05) - mousePos;
+        vec3 dir = normalize(seg);
+        float dist = length(seg);
+        if (dist < 1.5){
+          float force = clamp(1.0 / (dist * dist), -0., .2);
+          transformed += dir * log(force) * .25;
+          vNormal = force /0.5;
+        }
+      `
+    )
+}
+  const pointsMesh = new THREE.Points(modelGeo, pmaterial)
```

Comment:
- To keep the code organize?  Maybe can you move this code to the `models` folder at `Experience` folder.  Try to generate a reusable model. Maybe In the future I want 2 faces instead of 1.  To check if your solution work, try to print 2 models.

- Also please write the credits of the model. If it was just your talent make sure to write your name! Is your stuff ;-)  , if is not, try add some credit to whoever needed.

- Also can you verify why the colors are not really that nice... "SUCCESS" and "DANGER" are very bugy.

Reply:
![Screenshot 2024-01-19 at 17 20 52](https://github.com/tkalejandro/portfolio-sonia-coronado/assets/50468778/ab83d9d2-e23e-4b5e-8ea5-22f71e799e8e)

Reply:
Yes, success and danger colors are kinda white, if we try their darker variants they will work fine.
I am planning to make another ticket to improve the color display and make it vary based on the face part.

For example, hair will have a darker variant of success color, etc
![IMG_0355](https://github.com/tkalejandro/portfolio-sonia-coronado/assets/99765449/d6918027-52f9-4033-94a8-862c17757676)